migrations/0038-update-browsePage-slug.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+
+module.exports = function(migration) {
+  const browsePage = migration.editContentType('browsePage');
+  browsePage
+    .editField('identifier')
+    .required(true);
+
+  if (!process.env.SLUG_VALIDATION_APP_ID) {
```

Comment:
this could even go up before anything else is even initialized. No point in going on if the APP isn't set.